src/Campaigns/resources/admin/constants/goalInputAttributes.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * WordPress Dependencies
+ */
+import {__, sprintf} from '@wordpress/i18n';
+
+/**
+ * Internal Dependencies
+ */
+import {GoalInputAttributes} from '@givewp/campaigns/admin/components/CampaignFormModal/types';
+import {amountFormatter} from '@givewp/campaigns/utils';
+
+/**
+ * @unreleased
+ */
+export function getGoalInputAttributes(goalType: string, currency: string): GoalInputAttributes | undefined {
```

Comment:
I suggest adding a param type to `goalType` (amount, donations, etc.) either manually or you could move the attributes object outside and to `keyof typeof attributes` however that might be tricky with the dynamic currency formatter.

Reply:
Added shared type 066347f36e87e53249133cebd77ebfc6c18d4ddb